system/edit-config.in

```diff
@@ -13,6 +13,10 @@ fi
 [ -z "${NETDATA_USER_CONFIG_DIR}" ] && NETDATA_USER_CONFIG_DIR="@configdir_POST@"
 [ -z "${NETDATA_STOCK_CONFIG_DIR}" ] && NETDATA_STOCK_CONFIG_DIR="@libconfigdir_POST@"
 
+make_it_safer() {
+  find "${NETDATA_USER_CONFIG_DIR}"/*/ -type f ! -perm 640 -name "*.conf" -exec chmod o-rwx {} + -exec chown root:netdata {} +
```

Comment:
```suggestion
  find "${NETDATA_USER_CONFIG_DIR}"/* -type f ! -perm 640 -name "*.conf" -exec chmod o-rwx {} + -exec chown root:netdata {} +
```

Otherwise, it follows symlinks 🤷‍♂️ 

---

In theory, it is not always `netdata`, e.g. 

https://github.com/netdata/netdata/blob/92d48b177862e0b0d1957a0c97db80d4294e7884/packaging/docker/run.sh#L62

Reply:
I don't want netdata.conf to be impacted and this is exactly what the trailing slash does.
What kind of symbolic links you mean? There should not be any as far as I can see.
Also, you have not used sudo to run edit-config.sh.

Reply:
Yes, I think it is ok when I just need usage.

Reply:
> There should not be any as far as I can see.

```bash
$ pwd
/opt/netdata/etc/netdata
$ ls -l | grep orig
lrwxrwxrwx   35 root 13 May 13:26 orig -> /opt/netdata/usr/lib/netdata/conf.d
```

Reply:
> I don't want netdata.conf to be impacted and this is exactly what the trailing slash does.

Didn't get that from the code, that we just need to add additional ` ! -name "netdata.conf"`.

See, I have only `python.d/example.conf`

 - w/ `/`

```cmd
$ find /opt/netdata/etc/netdata/*/ -type f ! -perm 640 -name "*.conf" ! -name "netdata.conf" | wc -l
231
```

- w/o `/`

```cmd
$ find /opt/netdata/etc/netdata/* -type f ! -perm 640 -name "*.conf" ! -name "netdata.conf" | wc -l
1
```